release/models/platform/openconfig-platform-transceiver.yang

```diff
@@ -451,9 +457,11 @@ revision "2023-08-30" {
 
     container thresholds {
       description
-        "Enclosing container for transceiver alarm thresholds.
-        Each threshold is compared to the instant value of the
-        measured parameter corresponding to the threshold";
+        "Enclosing container for transceiver alarm thresholds. Each threshold
+        is compared to the instant value of the measured parameter
+        corresponding to the threshold. For optical input power, the
+        threshold applies only to the physical-channel(s).  This threshold
+        does not apply to the aggregate transceiver input-power.";
```

Comment:
should these messages be on each of the specific leaves instead? It is perhaps redundant, but seems more explicit and if we add something like aggregate power we won't have to remember to update the top level description?
https://github.com/openconfig/public/blob/5d24889a323a84fd077cbb83c2c7b5fb4896a6fe/release/models/platform/openconfig-platform-transceiver.yang#L794

Also, this should be for input and output.

Reply:
done